content/blog/2019-10-21-rmangal.md

```diff
@@ -0,0 +1,154 @@
+---
+slug: rmangal
+title: 'rmangal: making ecological networks easily accessible'
+date: 2019-10-21
+authors:
+  - Kevin Cazelles
+  - Steve Vissault
+categories: blog
+topicid: 1848
+tags:
+  - Software Peer Review
+  - R
+  - packages
+  - open-data
+  - data
+  - data-access
+  - ecology
+  - ecological networks
+  - API
+  - API client
+  - Mangal
+  - rmangal
+---
+
+
+In early September, the version 2.0.0 of [rmangal][rmangal] was approved by
+rOpenSci, four weeks latter it made the CRAN. Following-up on our experience we
+detail below the reasons why we wrote rmangal, why we submitted our package to
+rOpenSci and how the peer-review improve our package.
+
+
+
+### [Mangal](https://mangal.io/#/), a database for ecological networks
+
+Ecological networks are defined as a set of species populations (the nodes of
+the network) connected through ecological interactions (the edges). Interactions
+are ecological processes in which one species affects another. Although
+predation is probably the most known and documented interaction, other less
+noticeable associations are just as essential to ecosystem functioning. For
+instance, a mammal that unintentionally disperses viable seeds attached to his
+fur might help plants to thrive. All of these interactions occur simultaneously,
+shaping ecosystems functioning and making them as complex as they are
+fascinating.
+
+Recording and storing properly these interactions help ecologists to better
+understand ecosystems. That is why they are currently compiling datasets to
+explore how species associations vary over environmental gradients and how
+species lost might affect ecosystems functioning. This fundamental research
+question should help us understanding how ecological networks will respond to
+global change. To this end, the Mangal project <https://mangal.io/#/>
+standardizes ecological networks and eases their access. Every dataset contains
+a collection of networks described in a specific reference (a scientific
+publication, a book, etc.). For every network included in the database, Mangal
+include all the species name and several taxonomic identifiers
+([gbif](https://www.gbif.org/), [eol](https://eol.org/),
+[tsn](https://www.itis.gov/) etc.) as well as all interaction and their type.
+Currently, Mangal includes 172 datasets, which represents over 1300 ecological
+networks distributed worldwide.
+
+
+
+### An R client to make ecological networks easily accessible
+
+In 2016, the first paper describing the project was published[^original_ref]. In
+2018, a substantial effort was put in order to improve the data structure and
+gather new networks from existing publications. In 2019, the web API was
+rewritten, a new website launched and hundreds of new interactions were added.
+
+{{< figure src="/img/blog-images/2019-10-22-rmangal/mangal_explorer.png"
```

Comment:
This image is great in that it gives a glimpse of what the tool gives a user. However, it made me crave the ability to zoom in and explore. Any way you can link to that specific example in the site https://mangal.io/#/network? Would help readers quickly get a deeper understanding.

At the end of the section "Mangal, a database for ecological networks" I got very curious about having a specific example, so maybe there's an opportunity to write ~2 lines about an example from one dataset and link to that network in the site. That might be powerful.

Reply:
sounds good

Reply:
Fix in ab660256e07bf53e9726ed34fc3128e708d4f516

Reply:
We enhanced the post with a quick example, see ae9added2f15ba713e5305c3b07fe1815535ce87